components/centraldashboard/public/components/manage-users-view_test.js

```diff
@@ -0,0 +1,213 @@
+/* eslint-disable max-len */
+import '@polymer/test-fixture/test-fixture';
+import 'jasmine-ajax';
+import {mockIronAjax, yeildForRequests} from '../ajax_test_helper';
+import {flush} from '@polymer/polymer/lib/utils/flush.js';
+
+import './dashboard-view';
+
+const FIXTURE_ID = 'manage-users-view-fixture';
+const MU_VIEW_SELECTOR_ID = 'test-manage-users-view';
+const TEMPLATE = `
+<test-fixture id="${FIXTURE_ID}">
+  <template>
+    <manage-users-view id="${MU_VIEW_SELECTOR_ID}"></manage-users-view>
+  </template>
+</test-fixture>
+`;
+const TEST_USER = 'test@kubeflow.org';
+const newNS = (namespace, role, user='test@kubeflow.org') => ({role, namespace, user});
```

Comment:
This function feels a bit unnecessary since it's just shaping the arguments into a different object. For test-comprehension purposes I would just explicitly define the objects in the constants below.

Reply:
I realized I needed to add these as I was stepping through the tests. Seems like the update from the Ajax response to the view is not synchronous and the `flush()` call does not yield a tick for the async backlog to finish.